catalog/ui/search-ui/standard/server-impl.js

```diff
@@ -18,7 +20,25 @@ var URL = require('url'),
 var server = {};
 
 server.requestProxy = function (req, res) {
-    req.pipe(request("http://localhost:8181" + req.url)).pipe(res);
```

Comment:
Just change this line to use https and 8993 and put:
process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
above the line. Then you can remove all the stuff added below.